### ISSUE TITLE
spinner arrow not showing

### ISSUE THREAD

**Issue by earamsey** [skydoves/PowerSpinner#121]
**Please complete the following information:**
- Library Version - 1.2.3
- Affected Device(s) Galaxy S20+
 
**Describe the Bug:**

The spinner arrow is not showing. Here is my XML code;
```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:id="@+id/root"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:gravity="center"
    android:orientation="horizontal"
    android:weightSum="1">

    <RelativeLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:background="@drawable/border_box_bg"
        android:paddingBottom="20dp">

        <TextView
            android:id="@+id/title_tv"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_alignParentStart="true"
            android:layout_alignParentEnd="true"
            android:layout_marginLeft="@dimen/boxin_margin_bigger"
            android:layout_marginTop="10dp"
            android:layout_marginRight="@dimen/boxin_margin_bigger"
            android:layout_marginBottom="8dp"
            android:gravity="center_vertical|start"
            android:padding="8dp"
            android:paddingStart="0dp"
            android:paddingEnd="0dp"
            android:text="Choose a device"
            android:textColor="@android:color/black"
            android:textSize="16sp"
            tools:ignore="HardcodedText" />

        <Button
            android:id="@+id/push_btn"
            android:layout_width="match_parent"
            android:layout_height="48dp"
            android:layout_below="@id/title_tv"
            android:layout_marginStart="16dp"
            android:layout_marginEnd="16dp"
            android:background="@drawable/zzz_factor_btn_bg"
            android:gravity="center"
            android:paddingLeft="@dimen/_24sdp"
            android:paddingTop="8dp"
            android:paddingRight="@dimen/_24sdp"
            android:paddingBottom="8dp"
            android:text="@string/send_me_a_push"
            android:textAllCaps="false"
            android:textColor="@android:color/white"
            android:textSize="@dimen/_14sdp"
            tools:ignore="TextContrastCheck" />

        <Button
            android:id="@+id/call_btn"
            android:layout_width="match_parent"
            android:layout_height="48dp"
            android:layout_below="@id/push_btn"
            android:layout_marginStart="16dp"
            android:layout_marginTop="8dp"
            android:layout_marginEnd="16dp"
            android:background="@drawable/zzz_factor_btn_bg"
            android:gravity="center"
            android:paddingLeft="@dimen/_24sdp"
            android:paddingTop="8dp"
            android:paddingRight="@dimen/_24sdp"
            android:paddingBottom="8dp"
            android:text="@string/call_me"
            android:textAllCaps="false"
            android:textColor="@android:color/white"
            android:textSize="@dimen/_14sdp"
            tools:ignore="TextContrastCheck" />

        <TextView
            android:id="@+id/label_tv"
            android:layout_width="wrap_content"
            android:layout_height="48dp"
            android:layout_below="@id/call_btn"
            android:layout_marginStart="16dp"
            android:layout_marginTop="12dp"
            android:layout_marginEnd="8dp"
            android:gravity="center"
            android:text="@string/device"
            android:textColor="@android:color/black"
            android:textSize="16sp" />

        <com.skydoves.powerspinner.PowerSpinnerView
            android:id="@+id/spinner_btn"
            android:layout_width="match_parent"
            android:layout_height="48dp"
            android:layout_below="@id/call_btn"
            android:layout_alignEnd="@id/call_btn"
            android:layout_marginTop="12dp"
            android:layout_toRightOf="@id/label_tv"
            android:background="@drawable/border_box_green_bg"
            android:contentDescription="@string/device_selection_dropdown"
            android:gravity="center"
            android:paddingEnd="10dp"
            android:textColor="@android:color/black"
            android:textSize="14sp"
            app:spinner_arrow_animate="true"
            app:spinner_arrow_animate_duration="@android:integer/config_mediumAnimTime"
            app:spinner_arrow_drawable="@drawable/ic_spinner_arrow"
            app:spinner_arrow_gravity="end"
            app:spinner_arrow_show="true"
            app:spinner_arrow_tint="@color/Title"
            app:spinner_divider_color="@android:color/white"
            app:spinner_divider_show="true"
            app:spinner_divider_size="0.4dp"
            app:spinner_item_height="46dp"
            app:spinner_popup_animation="dropdown"
            app:spinner_popup_background="@drawable/border_box_bg"
            app:spinner_popup_elevation="14dp" />

    </RelativeLayout>
</LinearLayout>
```

**Expected Behavior:**

It is expected that the spinner arrow will be shown and is green using @drawable/ic_spinner_arrow for arrow. I also tried using default arrow and it does not appear either.

**Comment by skydoves** [comment]
Hey, could you change the color of the `@Color/Title`? Thanks!

**Comment by ar5-rehman** [comment]
@skydoves the color changes is not working, showing only the default color which is white, can you look into it?

**Comment by skydoves** [comment]
Thanks for reporting this issue! You can available it now in `1.2.4-SNAPSHOT`, and the fix will be included in the next stable release.

**Comment by skydoves** [comment]
`1.2.4` stable has been released. Thanks!
